ray-operator/DEVELOPMENT.md

```diff
@@ -97,6 +99,24 @@ kubectl logs {YOUR_OPERATOR_POD} | grep "Hello KubeRay"
 * The command `make docker-build` (Step 3) will also run `make test` (unit tests).
 * Step 6 also installs the custom resource definitions (CRDs) used by the KubeRay operator.
 
+#### Run the operator outside the cluster
+
+> Note: Running the operator outside the cluster allows you to debug the operator using your IDE. For example, you can set breakpoints in the code and inspect the state of the operator.
+
+```bash
+# Step 1: Create a Kind cluster
+kind create cluster --image=kindest/node:v1.24.0
```

Comment:
We should refernence newer kind versions in these docs, v1.24 is pretty old and not supported anymore

Reply:
Maybe this can be done in another PR? I simply copy the version from here.

https://github.com/ray-project/kuberay/blob/6186a7dedaef8e301e7013fef0a868be3b4b3295/ray-operator/DEVELOPMENT.md?plain=1#L64